ci/test_complete.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+
+# Compares options in `rg --help` output to options in zsh completion function
+
+set -e
+
+main() {
+    local rg="target/${TARGET}/release/rg"
```

Comment:
We probably can't do this exact thing. This would at, the very least, depend on ripgrep being built before these tests are run. I think we could probably do it by using a debug build, but right now, a release build is not built in CI. It might be easier to just use `grep`. :P (I think you could set `rg=grep` and it will just work.)